### PR TITLE
Add missed namelists

### PR DESCRIPTION
**Pull request by weiwangncar**
TYPE: no impact, text only

KEYWORDS: namelists, README

SOURCE: internal, Will Hatheway

DESCRIPTION OF CHANGES:
Problem:
A few newly added as well as a few pre-existing namelists are not in the README file. 

Solution:
Add these namelists in run/README.namelist file. Also make some lines in the file shorter for easy raed.

LIST OF MODIFIED FILES: 
M     run/README.namelist

TESTS CONDUCTED: 
1. Text only change, no tests required.

### CODE REVIEW

**Comment by mgduda** [comment]
@weiwangncar I think the contributor's name is "Will Hatheway" rather than "Will Haythway".

**Comment by weiwangncar** [comment]
> @weiwangncar I think the contributor's name is "Will Hatheway" rather than "Will Haythway".

Corrected. Thanks.

**Comment by weiwangncar** [comment]
@smileMchen @kkeene44 Can you review this PR? If you find more to add, let me know too.

File: run/README.namelist [hunk 1]
```diff
@@ -77,7 +77,8 @@ Description of namelist variables
  adjust_output_times                 = .false., ! adjust output times to the nearest hour
  override_restart_timers             = .false., ! whether to change the alarms from what is previously set
  write_hist_at_0h_rst                = .false., ! whether to output history file at the start of restart run
- output_ready_flag		     = .true.,  ! asks the model to write-out an empty file with the name 'wrfoutReady_d<domain>_<date>.  
+ output_ready_flag		     = .true.,  ! asks the model to write-out an empty file with the name 
+                                                  'wrfoutReady_d<domain>_<date>.
```
Comment (kkeene44) [inline run/README.namelist:81]: What is the purpose of moving these to the next line? I know it helps with wrapping, but that will still be dependent on the user's window. If I decrease the size of this window, even this begins to wrap.

File: run/README.namelist [hunk 2]
```diff
@@ -504,22 +508,32 @@ Namelist variables for controlling the adaptive time step option:
...
                                      = 50, P3 1-ice category, 1-moment cloud water
                                      = 51, P3 1-ice category plus double-moment cloud water
                                      = 52, P3 2-ice categories plus double-moment cloud water
                                      = 53, P3 1-ice category, 3-moment ice, plus double-moment cloud water
-                                     = 55, Jensen-ISHMAEL (Ice-Spheroids Habit Model with Aspect-ratio Evolution) scheme                                            predicting qc, qr, and three ice species with four predictive equations 
-                                           for each ice
+                                     = 55, Jensen-ISHMAEL (Ice-Spheroids Habit Model with Aspect-ratio Evolution) scheme 
+                                           predicting qc, qr, and three ice species with four predictive equations for each ice
                                      = 56, NTU multi-moment scheme  ! for ntu3m
+ ccnty                                 = 1  ! for aerosol background type, marine, for ntu3m
```
Comment (kkeene44) [inline run/README.namelist:532]: This option and description already exists further down in the file (line 561).

File: run/README.namelist (hunk 1 above)
Comment (weiwangncar) [reply run/README.namelist:81]: @kkeene44 True, but I just like to make the width of the text similar inside the file.

File: run/README.namelist (hunk 2 above)
Comment (weiwangncar) [reply run/README.namelist:532]: @kkeene44 Thanks for the file. I will select some to add. If you see any that may appear important to you or users, let me know.

Comment (weiwangncar) [reply run/README.namelist:538]: @kkeene44 I moved ccnty from below to the current location near the option it is supposed to be associated with.

**Review by smileMchen — APPROVED** [review]

**Review by kkeene44 — APPROVED** [review]

**Pull request merged**